CryptoExchange.Net.UnitTests/SocketClientTests.cs

```diff
@@ -183,5 +183,51 @@ public void FailingToConnectSocket_Should_ReturnError()
             // assert
             Assert.IsFalse(connectResult.Success);
         }
+
+        [TestCase()]
+        public async Task Error_response_should_not_confirm_subscription()
```

Comment:
Small request; can you update the test names to be in the same style as the other tests? So Xxxxx_Should_Yyyyyy